samples/sgx-ra/wasm-app/main.c

```diff
@@ -12,24 +12,43 @@
 int
 main(int argc, char **argv)
 {
-    char *evidence_json = NULL;
-    const char *hash = "12345678123456781234567812345678";
-    evidence_json = librats_collect((const uint8_t *)hash);
-    if (evidence_json == NULL) {
-        printf("Librats collect evidence failed.\n");
-        return -1;
+    // If you want to declare the evidence of type rats_sgx_evidence_t on the
+    // stack, you should modify the stack size of the CMAKE_EXE_LINKER_FLAGS in
+    // CMakeLists.txt to 51200 at least.
+    rats_sgx_evidence_t *evidence =
+        (rats_sgx_evidence_t *)malloc(sizeof(rats_sgx_evidence_t));
+    if (!evidence) {
+        printf("ERROR: No memory to allocate.\n");
     }
-    printf("evidence json:\n%s\n", evidence_json);
 
-    if (librats_verify(evidence_json, (const uint8_t *)hash) != 0) {
-        printf("Evidence is not trusted.\n");
+    // Generate user_data by SHA256 buffer and the wasm module.
+    // user_data = SHA256(buffer || sha256_wasm_module)
+    const char *buffer = "This is a sample.";
+
+    int ret_code = librats_collect(evidence, buffer);
+    if (ret_code != 0) {
+        printf("ERROR: collect evidence failed, %#x", ret_code);
+    }
+
+    // You could use these parameters for further verification.
+    printf("User Data: %s\n", evidence->user_data);
+    printf("MRENCLAVE: %s\n", evidence->mr_enclave);
+    printf("MRSIGNER: %s\n", evidence->mr_signer);
```

Comment:
The output of the execution is the following:

> ❯ ./iwasm wasm-app/test.wasm
> User Data: �|�
>               �����0,u���j��f
> MRENCLAVE: ��m��ږ�f��   L�ee�m��G�����z��
> MRSIGNER: ����}���p���*MwC�ș�i��p���Ξ
> Product ID: 0
> Security Version: 0
> Attributes.flags: 5
> Attribute.xfrm: 5
> Evidence is trusted.

Should we convert the content of these arrays into bytes when printing their content (for the demo)?